### PR TITLE
adding inital code to execute javascript, also fixed bug where not all elements are being downloaded due to different tag not caught in htmp parser

### PR DESCRIPTION
**Pull request by Devin-Simpson**

### CODE REVIEW

File: clientTest/clientTest.go [hunk 1]
```diff
@@ -42,56 +48,105 @@ func RunClientSideTest(res *http.Response, client *http.Client, wg *sync.WaitGro
...
+
+	return assetUrl
 }
+
+func LogElapstedTime(clientSideTime time.Time, totalClientTime float64, logMessage string, DETAILED_LOGGING bool) float64{
+
+	endClientSideTime := time.Now().Sub(clientSideTime)
+	totalClientTime = totalClientTime + endClientSideTime.Seconds()
+
+	if DETAILED_LOGGING {
```
Comment (tejom) [inline clientTest/clientTest.go]: does changing the package to main let this be a  global variable that doesnt need to be passed around?

Comment (Devin-Simpson) [reply clientTest/clientTest.go]: when I set the package of clientTest to main I get this error when running 'go build'

can't load package: /Users/dsimpso/mygo/src/github.com/tejom/load_histogram/load_histogram.go:18:2: import "github.com/tejom/load_histogram/clientTest" is a program, not an importable package

Comment (tejom) [reply clientTest/clientTest.go]: clientTest.go needs to be moved to the same directory ad loadhistogram.go. The package changed to main. the "clientTest." prefix to the methods removed and the variable can be shared between the files. 

I can do this after you merge this so it doesnt mess up all the work?

Comment (Devin-Simpson) [reply clientTest/clientTest.go]: ok, changes are made, it works

**Pull request merged**